src/mixins/view.js

```diff
@@ -163,11 +167,18 @@ export default {
     // https://github.com/jashkenas/backbone/blob/1.2.3/backbone.js#L1235
     this._removeElement();
 
+    if (shouldTriggerDetach) {
+      this._isAttached = false;
+      this.triggerMethod('detach', this);
+    }
+
     // remove children after the remove to prevent extra paints
     this._removeChildren();
 
     this._destroyBehaviors(args);
 
+    this._isDestroyed = true;
```

Comment:
Hmm, I will have to think more about this.  Maybe we can discuss on the call.  My first impression is that a view is not fully destroyed until its children and behaviors are gone as well.  If knowledge of "event in progress" is desired, setting a `this.isEventHappening = true` in user code within `onBeforeEvent()` could become a generalized recommendation to our users.